test/functional/wallet_txn_doublespend.py

```diff
@@ -46,7 +46,7 @@ def run_test(self):
         # blockchain sync later in the test when nodes are connected, due to
         # timing issues.
         for n in self.nodes:
-            assert n.getblockchaininfo()["initialblockdownload"] == False
+            assert_equal(n.getblockchaininfo()["initialblockdownload"], False)
```

Comment:
Seems there are too many ways to say the same thing. I think we shouldn't be adding  even more.

For boolean values, if the assertion fails, it is clear what value the bool was. There is no need to print the value and a constant.

Reply:
>Seems there are too many ways to say the same thing. I think we shouldn't be adding even more.

Right, in this case, I use the `assert_equal` but I could use `assert_true`, and the reason that I used this is only for consistency across error messages and source code. However, if you like I can restore the pure `assert`

Reply:
@MarcoFalke: What is your preference for what @vincenzopalazzo does here?

Reply:
> For boolean values, if the assertion fails, it is clear what value the bool was. There is no need to print the value and a constant.

I'd agree in a statically typed environment, but in Python that doesn't really hold? I think there's merit in always printing the value, to more quickly debug cases where e.g. the type of a response changes or becomes None/empty string?